cou/cli.py

```diff
@@ -134,20 +135,25 @@ def normal(text: str) -> str:
     )
 
 
-async def apply_plan(upgrade_plan: UpgradeStep) -> None:
+async def apply_plan(upgrade_plan: UpgradeStep, non_interactive: bool) -> None:
     """Apply the plan for upgrade.
 
     :param upgrade_plan: Plan to be executed on steps.
     :type upgrade_plan: UpgradeStep
     """
+    if non_interactive:
+        await upgrade_plan.run()
+        for sub_step in upgrade_plan.sub_steps:
+            await sub_step.run()
+        return
     result = "X"
     while result.casefold() not in AVAILABLE_OPTIONS:
         result = input(prompt(upgrade_plan.description)).casefold()
```

Comment:
not-blocking: What about
```suggestion
        result = "c" if non_interactive else input(prompt(upgrade_plan.description)).casefold()
```
This will be more readable to me and we will not define logic of run and iteration in two places.